mplc/experiment.py

```diff
@@ -84,11 +87,18 @@ def __init__(
                            a number of non-deterministic phenomena). Example: 5
         :param scenarios_list: list, list of scenarios to be run during the experiment.
                                Scenario can also be added via the .add_scenario() method.
+        :param is_save: boolean. If set to True, the experiment will be save on disk.
```

Comment:
Cool! @arthurPignet don't forget to update the parameters in the `documentation.md` accordingly!